packages/calypso-e2e/package.json

```diff
@@ -21,10 +21,18 @@
 	},
 	"devDependencies": {
 		"@types/config": "^0.0.38",
-		"@types/node": "^15.0.2"
+		"@types/jest": "^25.2.3",
+		"@types/node": "^15.0.2",
+		"jest": "^26.6.3",
+		"ts-jest": "^26.5.6"
 	},
 	"scripts": {
 		"clean": "yarn build --clean && npx rimraf dist",
-		"build": "tsc --build ./tsconfig.json"
+		"build": "tsc --build ./tsconfig.json",
+		"test": "jest"
+	},
+	"jest": {
```

Comment:
Packages in the monorepo are expected to implement tests by having a `jest.config.js` file that extends from `../../test/packages/jest-preset.js` (plus any needed customization).

They also don't need to depend on `jest` or implement a script `test`, as testing is run using the scripts `yarn test-packages ...` declared in the root of the project.

Reply:
I admit I was not aware of this, I will look for other examples under `packages` and follow them.

Reply:
Probably is not documented anywhere 😬